contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -292,7 +292,7 @@ GRANT SELECT ON sys.babelfish_authid_user_ext TO PUBLIC;
 -- DATABASE_PRINCIPALS
 CREATE VIEW sys.database_principals AS SELECT
```

Comment:
Let's change this into CREATE OR REPLACE VIEW, same applies for server_principals and sp_statistics_view